judgments/utils.py

```diff
@@ -212,4 +214,7 @@ def get_judgment_by_uri(judgment_uri: str) -> Judgment:
         use_https=settings.MARKLOGIC_USE_HTTPS,
     )
 
-    return Judgment(judgment_uri, api_client)
+    try:
+        return Judgment(judgment_uri, api_client)
+    except JudgmentNotFoundError:
+        raise Http404("Judgment not found")
```

Comment:
This doesn't feel like the right place to raise a 404 error; we might be using this utility for other things where we want JudgmentNotFound to bubble up.